packages/makeup-menu/src/index.js

```diff
@@ -181,7 +183,7 @@ function _selectMenuItemCheckbox(widgetEl, menuItemEl) {
   }
 }
 
-function _selectMenuItemRadio(widgetEl, menuItemEl) {
+function _selectMenuItemRadio(widgetEl, menuItemEl, options) {
   if (menuItemEl.getAttribute("aria-disabled") !== "true") {
```

Comment:
I realize this is not part of this PR, but while we're at it, we should change this too. Lengthy and nested conditional blocks should be avoided when possible. It improves readability and adds simplicity. 

So, this would turn into this:

```js
// if menu item element is disabled, exit function
if (menuItemEl.getAttribute("aria-disabled") === "true") {
  return;
}
```